### DISCUSSION TITLE
Can AzerothCore work with localized Wow client

### DISCUSSION THREAD

**Discussion by felixjiang** [azerothcore/azerothcore-wotlk#12424] [Guides & Tips]
Can the localized 3.3.5a Wow client work with AzerothCore?

How can the game objects (like gears, spells etc) displayed in localized text as well?

**Comment by virgo77** [comment]
Hi!
You can set the locale in the worldserver.conf file.
In linux, the file is in the 'etc' folder
In Windows, it's in the 'bin' folder (hope I'm not wrong)

At line 'around' 560, you can find a section named SERVER SETTINGS, where you can set:
- the Game Type (PvP / PvE / etc.)
- the RealmZone (China / Europe / Russia / Korea / etc.)
- the DBC Locale where you can set the language (around line 660). By default it is set to 255 to make the core autodetect the locale. But you can 'force' it in this file according to your language, or your country.

And in your client, you can set your language too. Just go in the WTF folder, and then open the config.wtf file. You will find a line with the language. Note that you need to download the client according to your language, as in the DATA folder you will find a folder named like your locale:
- frFR = France
- frCA = Canada
- enGB = United Kingdom
- enUS = USA
- esSP = Spanish
- and so go on...

Have fun!

V.

**Reply by felixjiang** [reply]
This is awesome. Thanks. I will try it this weekend
